Getting non-null price returns error

Trying to send an SMS internationally resulted in the following error when attempting to retrieve the price:

getObject is protected and there is no way to access it.

Caused by: java.lang.IllegalArgumentException: Property price is an object. Use getObject() instead.
at com.twilio.sdk.resource.InstanceResource.getProperty(InstanceResource.java:70)
at com.twilio.sdk.resource.instance.Sms.getPrice(Sms.java:172)
at com.xoom.sms.twilio.transform.TwilioSmsResourceTransformer.getPrice(TwilioSmsResourceTransformer.java:32)
at com.xoom.sms.twilio.transform.TwilioSmsResourceTransformer.transform(TwilioSmsResourceTransformer.java:25)
at com.xoom.sms.service.send.SmsTemplateProcessingSendingService.send(SmsTemplateProcessingSendingService.java:42)
at com.xoom.sms.service.notification.XferSmsNotificationServiceImpl.sendSms(XferSmsNotificationServiceImpl.java:59)
at com.xoom.sms.service.notification.XferSmsNotificationServiceImpl.notify(XferSmsNotificationServiceImpl.java:50)

--- a/src/main/java/com/twilio/sdk/resource/InstanceResource.java
+++ b/src/main/java/com/twilio/sdk/resource/InstanceResource.java
@@ -75,6 +75,12 @@ public abstract class InstanceResource extends Resource {
 				+ " is an object.  Use getObject() instead.");
 	}
 
+  /**
+   * Gets the property as an Object.
+   *
+   * @param name the name of the property
+   * @return the property as an Object
+   */
 	public Object getObject(String name) {
 	    	Object prop = getAndLoadIfNecessary(name);
 

